.circleci/config.yml

```diff
@@ -16,43 +15,7 @@ workflows:
               ignore: /.*/
 
 jobs:
-  ics721:
```

Comment:
Why are we knocking out ics721 cargo + lint job entirely? Is this a temporary stop gap?

Reply:
It's there, see the rest of the file.